distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/ContainerPerClassItestSuite.java

```diff
@@ -22,5 +21,5 @@
  * order as they appear in the array.
  */
 @RunWith(Suite.class)
-@Suite.SuiteClasses({TestEmbeddedSolr.class})
+@Suite.SuiteClasses({})
```

Comment:
❓ No issue with having an empty suite? Does this pass at runtime okay? Does it still spin up the container and waste that time?

Reply:
This doesn't spin up the container so we save the time. It hasn't shown any issues running either.